content/dstu2/allergy-intolerance.md

```diff
@@ -84,7 +84,7 @@ To successfully POST an allergy, the following headers must be provided. Allergy
     access-control-max-age → 0
     cache-control → no-cache
     etag → W/"35502191"
-    location → https://fhir-ehr.sandboxcerner.com/dstu2/d075cf8b-3261-481d-97e5-ba6c48d3b41f/AllergyIntolerance/35502191
+    location → https://fhir-ehr.sandboxcerner.com/dstu2/0b8a0111-e8e6-4c26-a91c-5069cbc6b1ca/AllergyIntolerance/35502191
```

Comment:
The goal was to fix the URLs. It looks like I missed this one and the patient search